src/components/footer/footer.ts

```diff
@@ -13,15 +14,16 @@ export class FooterComponent extends LitElement {
     const { currentYear, STARTING_YEAR } = this;
 
     return html`
-      <footer class="container as-footer">
-        <div class="row">
-
-          <div class="col-xs-12">
-            <p class="copyright-text">&copy; ${STARTING_YEAR} - ${currentYear} Analog Studios</p>
+      <footer class='container as-footer'>
+        <div class='row'>
+          <div class='col-xs-12'>
+            <p class='copyright-text'>&copy; ${STARTING_YEAR} - ${currentYear} Analog Studios</p>
+          </div>
+          <div class='col-xs-12'>
+            <app-socials-media></app-socials-media>
```

Comment:
I'm thinking maybe it would be better to put these links above the copyright?  Also maybe making them a little bigger?